### PR TITLE
Upgrade rush to 5.113.4

### PR DESCRIPTION
**Pull request by mgamis-msft**
# What
Upgrade rush to 5.113.4

# Why
OCE task

# How Tested
Ran `rush update` and then `rush build:all-flavors` as directed in https://skype.visualstudio.com/SPOOL/_wiki/wikis/SPOOL.wiki/47006/Updating-Dependencies?anchor=update-rush-if-required

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: rush.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush.schema.json",
-  "rushVersion": "5.112.2",
+  "rushVersion": "5.113.4",
```
Comment (emlynmac) [inline rush.json:3]: I noticed the `breaking-change` label is set on this PR. 
Why is this necessary?

Comment (mgamis-msft) [reply rush.json:3]: You're right it's not necessary. Removed.
